### ISSUE TITLE
Modify the benchmark output file name with hours, minutes information

### ISSUE THREAD

**Issue by oldmikeyang** [intel-analytics/ipex-llm#11406]
The current all-in-one benchmark save the csv file with the name only have date.
if we run multi test in the same day, the older test data will be overwrite by the new one.

Suggest to modify the file name with hours and minutes.
Here is the suggested code change, and the new file name example

../../../dev/benchmark/all-in-one/pipeline_parallel_gpu-results-2024-06-22_22-20-51.csv

```
diff --git a/python/llm/dev/benchmark/all-in-one/run.py b/python/llm/dev/benchmark/all-in-one/run.py
index 2036f6f9ee..df7ae3539e 100644
--- a/python/llm/dev/benchmark/all-in-one/run.py
+++ b/python/llm/dev/benchmark/all-in-one/run.py
@@ -25,6 +25,7 @@ import csv

 import numpy as np
 from datetime import date
+import datetime

 import os
 current_dir = os.path.dirname(os.path.realpath(__file__))
@@ -1826,7 +1827,9 @@ def run_pipeline_parallel_gpu(repo_id,
 if __name__ == '__main__':
     from omegaconf import OmegaConf
     conf = OmegaConf.load(f'{current_dir}/config.yaml')
-    today = date.today()
+    # today = date.today()
+    now = datetime.datetime.now()
+    today=now.strftime("%Y-%m-%d_%H-%M-%S")
     if 'exclude' in conf:
         excludes = conf['exclude']
     streaming = False

```

**Comment by qiuxin2012** [comment]
This change will affect our nightly performance test's post data processing. @lalalapotter will look into this.

**Comment by lalalapotter** [comment]
Fixed in PR #11444 , please try it.
